packages/now-cli/src/index.js

```diff
@@ -688,8 +682,6 @@ const main = async argv_ => {
   return exitCode;
 };
 
-debug('start');
```

Comment:
This was dead code, since the `debug()` function is a no-op until the `main()` function is called.